.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -9,13 +9,13 @@ jobs:
   strategy:
     maxParallel: 8
     matrix:
-      gcc_6:
-        CXX_COMPILER: 'g++-6'
+      gcc_7:
+        CXX_COMPILER: 'g++-7'
         PYTHON_VER: '3.8'
-        C_COMPILER: 'gcc-6'
+        C_COMPILER: 'gcc-7'
         F_COMPILER: 'gfortran'
         BUILD_TYPE: 'release'
-        APT_INSTALL: 'gfortran gcc-6 g++-6'
+        APT_INSTALL: 'gfortran gcc-7 g++-7'
```

Comment:
also edit here https://github.com/psi4/psi4/blob/master/cmake/custom_cxxstandard.cmake#L11 , please. and if you feel like fixing the block above it by killing off c++14 std, go for it.

Reply:
I've tried to update all the compiler versions in `custom_cxxstandard.cmake`, so please let me know if I got anything wrong.